### PR TITLE
messages

### LINKED ISSUE #390: Reason message in each assert

### PR DESCRIPTION
**Pull request by l3r8yJ**
closes #390

@h1alexbel take a look, please

---

## PR-Codex overview
This PR focuses on improving the code readability and providing more descriptive error messages. 

### Detailed summary
- Improved readability by adding line breaks and indentation.
- Added descriptive error messages for better understanding of exceptions thrown.
- Refactored code to use formatted strings for better readability.
- Fixed imports and removed unused code.

> The following files were skipped due to too many changes: `src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java`, `src/test/java/io/github/eocqrs/kafka/fake/FkConsumerTest.java`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by zoeself** [comment]
@l3r8yJ thank you for your Pull Request. I'll assign someone to review it soon. 

 If this PR solves a ``todo`` from the code, please don't forget to remove it.

**Comment by zoeself** [comment]
@h1alexbel please review this Pull Request. Deadline (when it should be merged or closed) is ``2023-07-11T20:09:04.552438``.

You should check if the requirements have been implemented (partially or in full), if there are unit tests covering the changes and if the CI build passes. Feel free to reject the PR or ask for changes if it's too big or not clear enough.

Estimation here is ``30 minutes``, that's how much you will be paid. You will be paid even if this PR gets rejected.

**Comment by l3r8yJ** [comment]
@h1alexbel reminder

File: src/test/java/io/github/eocqrs/kafka/fake/FkConsumerTest.java [hunk 1]
```diff
@@ -386,15 +412,18 @@ void pollsRecords() throws Exception {
...
     final List<String> datasets = new ListOf<>();
     first.forEach(rec -> datasets.add(rec.value()));
+    final List<String> topics = List.of("test1", "test2", "test3");
     MatcherAssert.assertThat(
-      "First datasets in right format",
+      "Datasets %s should contain all of %s"
+        .formatted(datasets, topics),
       datasets,
-      Matchers.contains("test1", "test2", "test3")
+      Matchers.containsInAnyOrder(topics.toArray())
```
Comment (h1alexbel) [inline src/test/java/io/github/eocqrs/kafka/fake/FkConsumerTest.java:420]: @l3r8yJ I think that order is important here

**Comment by l3r8yJ** [comment]
@h1alexbel fixed, take a look, please 

need to merge before #410

**Comment by zoeself** [comment]
@h1alexbel Don't forget to close this ticket before the deadline (2023-07-11T20:09:05). You are past the first half of the allowed period.

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@rultor merge

**Comment by h1alexbel** [comment]
@l3r8yJ thanks

**Comment by zoeself** [comment]
@h1alexbel thank you for resolving this ticket. I've just added it to your active invoice. You can always check all your invoices and more on the [Contributor Dashboard](https://self-xdsd.com).

**Pull request merged**